src/Integration.UnitTests/LocalServices/RuleSetInspectorTests.cs

```diff
@@ -88,41 +89,38 @@ private string VsRuleSetsDirectory
         [TestMethod]
         public void RuleSetInspector_FindConflictingRules_ProjectLevelOverridesOfTheSolutionRuleset()
         {
-            using (var temps = new TempFileCollection())
-            {
-                // Setup
-                string solutionRuleSet = this.CreateCommonRuleSet().FilePath;
-                temps.AddFile(solutionRuleSet, false);
+            // Setup
+            string solutionRuleSet = this.CreateCommonRuleSet().FilePath;
+            this.temporaryFiles.AddFile(solutionRuleSet, false);
```

Comment:
There don't seem to be any occurrences where the created file isn't added to list of temporary files - why not push all of the .AddFile calls into the CreateXXX methods? (possibly renaming the helper methods to e.g. _CreateTempCommonRuleSet_, but I don't think that's necessary).